bee-common/bee-task/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "bee-task"
+version = "0.1.0"
+authors = [ "IOTA Stiftung" ]
+edition = "2018"
+description = "Task related utilities"
```

Comment:
add "." after sentence?

Reply:
We never do in Cargo.toml descriptions

Reply:
but the description could have more then 1 sentence? so you would need punctuation?

Reply:
I think the convention is no dot if you use `"` for the description and a dot if you use `"""`. I am happy having the discussion outside of this PR and change if we all agree but I wouldn't add the dot in this PR as it would be inconsistent.

Reply:
Okey understand - didn't know about the `"""`.